src/ps-web-apis.ts

```diff
@@ -53,7 +53,7 @@ export type FetchOptions = RequestInit & { timeout?: number };
  * Custom fetch interface which includes the possibility to customize timeouts for fetch requests
  */
 export type Fetch = (input: RequestInfo, init?: FetchOptions) => Promise<Response>;
-
+export type GetRosettaEnvByKey = (key: string) => string;
```

Comment:
should be:
```suggestion
export type GetRosettaEnvByKey = (key: string) => string | undefined;
```

as you can not trust rosetta to have always a value for given key

Reply:
Oh yeah totally blanked on that. Thanks